delivery_package_number/models/stock_picking_type.py

```diff
@@ -6,7 +6,12 @@
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
 
-    force_set_number_of_packages = fields.Boolean()
+    force_set_number_of_packages = fields.Selection(
+        [
+            ("always", "Always"),
+            ("partner", "Partner condition"),
+        ]
+    )
```

Comment:
You will need a migration script for this

Reply:
Yes! I know, i wrote in my comment